lib/utilities.py

```diff
@@ -0,0 +1,25 @@
+from datetime import datetime
+from dateutil.parser import parse
+import pytz
+
+def toUTC(suspectedDate, localTimeZone='UTC'):
+    '''make a UTC date out of almost anything'''
+
+    utc=pytz.UTC
+    objDate=None
+
+    if type(suspectedDate) == datetime:
+        objDate = suspectedDate
+    elif str(suspectedDate).isdigit():
+        # epoch? but seconds/milliseconds/nanoseconds (lookin at you heka)
+        epochDivisor = int(str(1) + '0'*(len(str(suspectedDate)) % 10))
```

Comment:
I removed a couple methods to detect if a variable is a digit, and to return the number of digits. I believe we don't need to support numbers like 1+2i do we?